eZ/Publish/Core/Persistence/Cache/Tests/UrlAliasHandlerTest.php

```diff
@@ -29,7 +29,7 @@ public function providerForUnCachedMethods(): array
         // string $method, array $arguments, array? $tags, array? $key, mixed? $returnValue
         return [
             ['publishUrlAliasForLocation', [44, 2, 'name', 'eng-GB', true, false], ['urlAlias-location-44', 'urlAlias-location-path-44', 'urlAlias-notFound']],
-            ['createCustomUrlAlias', [44, '1/2/44', true, null, false], ['urlAlias-location-44', 'urlAlias-location-path-44', 'urlAlias-notFound']],
```

Comment:
Not sure how those test are built, but should we not add new case instead of correcting old one to match new behavior? As from also I see on travis change per se didn't cause those to fail.